prototypes/malc0de.yml

```diff
@@ -0,0 +1,57 @@
+url: http://malc0de.com/bl/
+description: >
+  IP blacklist from malc0de DNS Sinkhole
+
+prototypes:
+  IP:
+    author: Pansaen Boonyakarn
+    development_status: EXPERIMENTAL
+    node_type: miner
+    indicator_types:
+      - IPV4
+    tags:
+      - OSINT
+      - ConfidenceHigh
+      - ShareLevelGreen
+    description: The files below will be updated daily with domains that have been indentified distributing malware during the past 30 days.
+    config:
+      source_name: malc0de.IP
+      attributes:
+        type: IPv4
+        direction: inbound
+        confidence: 70
+        share_level: green
+      ignore_regex: '[//S].*'
+      url: http://malc0de.com/bl/IP_Blacklist.txt
+      interval: 300
+      age_out:
+        default: null
+        sudden_death: true
+    class: minemeld.ft.http.HttpFT
+  DOMAINS:
+    author: Pansaen Boonyakarn
+    development_status: EXPERIMENTAL
+    node_type: miner
+    indicator_types:
+      - domain
+    tags:
+      - OSINT
+      - ConfidenceHigh
+      - ShareLevelGreen
+    description: This file will be automatically updated daily and populated with the last 30 days of malicious domains.
+    config:
+      source_name: malc0de.DOMAINS
+      attributes:
+        type: domain
+        direction: inbound
```

Comment:
This is not needed on domains indicators, `direction` is only valid for IPv4/IPv6 indicators